libs/api/src/lib/graphql/user.ts

```diff
@@ -170,20 +171,58 @@ export const GraphQLPublicUser = new GraphQLObjectType<UserWithRelations, Contex
     id: {type: GraphQLNonNull(GraphQLString)},
     name: {type: GraphQLNonNull(GraphQLString)},
     firstName: {type: GraphQLString},
-    email: {type: GraphQLNonNull(GraphQLString)},
-    preferredName: {type: GraphQLString},
-    address: {type: GraphQLUserAddress},
-    flair: {type: GraphQLString},
+    email: {
+      type: GraphQLNonNull(GraphQLString),
```

Comment:
I guess returns empty string to not have a breaking change? Could also make it nullable I guess, I don't mind either way.